code/network_analysis.py

```diff
@@ -0,0 +1,200 @@
+import project_config
+from __future__ import division
+from scipy import stats
+from general_utils import prepare_img_single, prepare_mask
+from os.path import join
+from pca_utils import first_pcs_removed
+import numpy as np
+import math
+import nibabel as nib
+import numpy.linalg as npl
+import roi_extraction
+import itertools
+
+
+file_name_con = "/Volumes/G-DRIVE mobile USB/fmri_con/sub011_task001_run001_func_data_mni.nii.gz"
+file_name_scz = "/Volumes/G-DRIVE mobile USB/fmri_scz/sub001_task001_run001_func_data_mni.nii.gz"
+img_con = nib.load(file_name_con)
+img_scz = nib.load(file_name_scz)
+data_con = img_con.get_data()[..., 5:]
+data_scz = img_scz.get_data()[..., 5:]
+
+def roi_cor (data, roi1,roi2):
+	"""
```

Comment:
Although Fisher's transformation is the method using in the paper, how to handle the situation when r closed to 1 is unclear, since this method is only robustly stabilize the variance when | r | is not too large and N is not too small.